src/Tizen.Applications.ComponentBased/Tizen.Applications.ComponentBased.Common/WidgetComponent.cs

```diff
@@ -0,0 +1,88 @@
+/*
+ * Copyright (c) 2021 Samsung Electronics Co., Ltd All Rights Reserved
+ *
+ * Licensed under the Apache License, Version 2.0 (the License);
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an AS IS BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+using System;
+
+namespace Tizen.Applications.ComponentBased.Common
+{
+    /// <summary>
+    /// The class for showing UI module
+    /// </summary>
+    /// <since_tizen> 9 </since_tizen>
+    public abstract class WidgetComponent : BaseComponent
+    {
+
+        /// <summary>
+        /// Overrides this method to handle behavior when the component is launched.
```

Comment:
Grammar issue, but you could reorganize the sentence. Please change it in the all documentation comments. "Overrides this method to" is kind of redundant, because that what you do with abstract and virtual methods.

```suggestion
        /// Override this method to handle behavior when the component is launched.
```

Reply:
Done. Thanks.